### ISSUE TITLE
ImportError: cannot import name 'pass_context' from 'jinja2.utils'

### ISSUE THREAD

**Issue by harshad16** [thoth-station/user-api#1701]
**Describe the bug**

User-api is failing in v0.34.10

```
2022-03-08 01:35:20,438  22 ERROR    thoth.user_api.openapi_server:1891: Exception on /api/v1/ui/ [GET]
Traceback (most recent call last):
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/opt/app-root/lib64/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/opt/app-root/lib64/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/opt/app-root/lib64/python3.8/site-packages/connexion/apis/flask_api.py", line 307, in console_ui_home
    return flask.render_template_string(index_path.read_text(), **template_variables)
  File "/opt/app-root/lib64/python3.8/site-packages/flask/templating.py", line 155, in render_template_string
    return _render(ctx.app.jinja_env.from_string(source), context, ctx.app)
  File "/opt/app-root/lib64/python3.8/site-packages/flask/helpers.py", line 957, in __get__
    value = self.func(obj)
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 689, in jinja_env
    return self.create_jinja_environment()
  File "/opt/app-root/lib64/python3.8/site-packages/flask/app.py", line 781, in create_jinja_environment
    rv = self.jinja_environment(self, **options)
  File "/opt/app-root/lib64/python3.8/site-packages/flask/templating.py", line 45, in __init__
    BaseEnvironment.__init__(self, **options)
  File "/opt/app-root/lib64/python3.8/site-packages/jinja2/environment.py", line 359, in __init__
    self.policies = DEFAULT_POLICIES.copy()
  File "/opt/app-root/lib64/python3.8/site-packages/jinja2/environment.py", line 108, in load_extensions
    environment: "Environment",
  File "/opt/app-root/lib64/python3.8/site-packages/jinja2/utils.py", line 131, in import_string
    " old name will be removed in Jinja 3.1.",
  File "/opt/app-root/lib64/python3.8/site-packages/jinja2/ext.py", line 18, in <module>
    from .utils import pass_context
ImportError: cannot import name 'pass_context' from 'jinja2.utils' (/opt/app-root/lib64/python3.8/site-packages/jinja2/utils.py)
```

**To Reproduce**
1. Go to 'https://stage.thoth-station.ninja/api/v1/ui/'

**Comment by mayaCostantini** [comment]
It looks like `flask==1.1.4` that we use is requires [`"Jinja2 >= 2.10.1, < 3.0"`](https://github.com/pallets/flask/blob/1ca199f9b38b70a4e97cb47a4252ffd7fccc008c/setup.py#L57) but `pass_context` was added in [`jinja2 3.0.0`](https://github.com/pallets/jinja/blob/681f8de8421183cf4ed35547c723cb0a3d9a2955/src/jinja2/utils.py#L40). Maybe we should consider using another version of `flask`.

**Comment by goern** [comment]
will this result in a new prescription?

**Comment by mayaCostantini** [comment]
@goern Yes, I will open a PR for this one soon 👍🏻

**Comment by mayaCostantini** [comment]
/close
Fixed by #1702
